helm/vitalam-ipfs/templates/_helpers.tpl

```diff
@@ -18,9 +18,21 @@ Create name to be used with deployment.
 Create chart name and version as used by the chart label.
 */}}
 {{- define "vitalam-ipfs.chart" -}}
-{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" | lower}}
 {{- end }}
 
+{{- define "vitalam-ipfs-node.fullname" -}}
+{{- if .Values.vitalamNode.enabled -}}
+{{- template "vitalam-node.fullname" .Subcharts.vitalamNode -}}
+{{- end -}}
+{{- end -}}
+
+{{- define "vitalam-ipfs.ipfsApiPort" -}}
+{{- if .Values.config.ipfsApiPort -}}
+{{- .Values.config.ipfsApiPort | quote -}}
+{{- end -}}
+{{- end -}}
+
```

Comment:
These are defined but never used? If they're used in a parent chart it should be used in this chart as well to ensure applied consistencvy right?

Reply:
they are both for hoisting, yes.

We could use the ipfsApiPort but we already refer to `.Values.config.ipfsApiPort` in the configmap.